openerp/openupgrade/doc/source/modules80-90.rst

```diff
@@ -15,542 +15,657 @@ Status :
-+-----------------------------------+-----------------------------------+
-|web_tip                            | Nothing to do                     |
-+-----------------------------------+-----------------------------------+
-|web_view_editor                    | Nothing to do                     |
-+-----------------------------------+-----------------------------------+
+Modules that are new in this release are marked with |new|. Modules that are
+missing in the new release are marked with |del|.
+
+.. |new| image:: images/new.png
+.. |del| image:: images/deleted.png
+
++----------------------------------------+-----------------------------------+
+|Module                                  |Status                             |
++========================================+===================================+
+|account                                 | Done (tentative)                  |
++----------------------------------------+-----------------------------------+
+|account_accountant                      | Nothing to do                     |
++----------------------------------------+-----------------------------------+
+| |del| account_analytic_analysis        |                                   |
++----------------------------------------+-----------------------------------+
+|account_analytic_default                | Nothing to do                     |
++----------------------------------------+-----------------------------------+
+| |del| account_analytic_plans           |                                   |
++----------------------------------------+-----------------------------------+
+| |del| account_anglo_saxon              |                                   |
++----------------------------------------+-----------------------------------+
+|account_asset                           | Done                              |
++----------------------------------------+-----------------------------------+
+| |del| account_bank_statement_extensions|                                   |
++----------------------------------------+-----------------------------------+
+| |new| account_bank_statement_import    |                                   |
++----------------------------------------+-----------------------------------+
+|account_budget                          | Nothing to do                     |
++----------------------------------------+-----------------------------------+
+|account_cancel                          | Nothing to do                     |
++----------------------------------------+-----------------------------------+
+| |del| account_chart                    |                                   |
++----------------------------------------+-----------------------------------+
+| |new| account_check_printing           | Done                              |
++----------------------------------------+-----------------------------------+
+| |del| account_check_writing            | Renamed [#account_check_writing]_ |
```

Comment:
Yes, I see that this a foot note, but at first sight I saw renamed and the same name and thought that there was an error.